libs/tools/barista/src/transform.ts

```diff
@@ -202,6 +202,31 @@ export const copyHeadlineTransformer: BaPageTransformer = async source => {
   return transformed;
 };
 
+/** Replaces internal href links with routerLink links */
+export const internalLinksToRouterLinksTransformer: BaPageTransformer = async source => {
+  const transformed = { ...source };
+  if (source.content && source.content.length) {
+    transformed.content = runWithCheerio(source.content, $ => {
+      const links = $('a');
+      if (links.length) {
+        links.each((_, link) => {
+          const linkValue = $(link).attr('href');
+          if (
+            linkValue &&
+            (!linkValue.startsWith('http') || !linkValue.includes('//:'))
```

Comment:
This check is rather complicated to read - I think what we want to do is check whether a link is relative or an absolute URL. 

I'd create a function that is called: isAbsoluteUrl(url: string): boolean

see https://stackblitz.com/edit/typescript-em8xdt

And we can then use this function in this expression